fe/fe-core/src/main/java/com/starrocks/connector/delta/DeltaLakeConnector.java

```diff
@@ -31,14 +34,18 @@ public class DeltaLakeConnector implements Connector {
 
     public static final String HIVE_METASTORE_URIS = "hive.metastore.uris";
     private final Map<String, String> properties;
+    private final CloudConfiguration cloudConfiguration;
     private final String catalogName;
     private final DeltaLakeInternalMgr internalMgr;
     private final DeltaLakeMetadataFactory metadataFactory;
 
     public DeltaLakeConnector(ConnectorContext context) {
         this.catalogName = context.getCatalogName();
         this.properties = context.getProperties();
-        this.internalMgr = new DeltaLakeInternalMgr(catalogName, properties);
+        this.cloudConfiguration = CloudConfigurationFactory.buildStorageCloudConfiguration(properties);
```

Comment:
There are few method to create `CloudConfiguration`, `HdfsEnvironment` and `Configuration`. Plz unify them.

Reply:
CloudConfiguration and HdfsEnvironment are two things, can't unify it.